tests/gdb-tests/tests/test_gdblib_parameter.py

```diff
@@ -26,6 +34,10 @@
 def test_gdb_parameter_default_value_works(start_binary, params):
     name_suffix, default_value, displayed_value, optional_kwargs = params
 
+    # GDB < 9.x does not support PARAM_ZUINTEGER_UNLIMITED
+    if name_suffix == "unlimited-zuint" and not optional_kwargs:
+        pytest.skip("GDB < 9.x does not support `gdb.PARAM_ZUINTEGER_UNLIMITED`")
```

Comment:
Maybe just: `if not params: pytest.skip("Current GDB version does not support this testcase")`?

Reply:
Yes, this looks better.
I'll change it this way.